src/components/molecules/Settings/NavigationItem/index.tsx

```diff
@@ -22,22 +27,18 @@ const NavigationItem: React.FC<Props> = ({ name, to, children }) => {
 
 const LinkItem = styled(Link)`
   display: flex;
-  padding: 16px;
-  color: #ffffff;
+  padding: 16px 8px;
```

Comment:
metrics?

Reply:
yes, but we really really need metrics to use in all padding and marging? like, we will not change any space when we switch the theme...  any way, it's really Crush me about using metrics 💀

Reply:
The idea behind the metrics (like colors, etc) is that we can change them in one place (the Theme files) and they'll change everywhere. Metrics is a little debatable though as changes to metrics will probably be accompanied by larger changes, whereas color changes are relatively minor and changing one source is really convenient.
But anyways, as much as possible we are trying to consistently make use of the available metrics.

Reply:
ok, thanks